spec/unit/actions/synchronize_users_spec.rb

```diff
@@ -44,8 +44,8 @@
         allow(File).to receive(:open).and_return("client_id\nclient123")
       end
 
-      it 'uses client_id column' do
-        expect(domain).to receive(:clients).with('client123')
+      it 'uses client_id column to fetch client' do
```

Comment:
Similar code found in synchronize_user_filters_spec.rb:53, synchronize_users_spec.rb:47 (mass = 32)